webapp/platform/types/src/client4.ts

```diff
@@ -14,12 +14,16 @@ export type ClientResponse<T> = {
     data: T;
 };
 
+/**
+  * @ignoreStatus boolean - If true, status codes > 300 are ignored and don't cause an error
```

Comment:
I think it might be better if you just add a JSDoc comment to just the `ignoreStatus` field instead of adding it to `Options`

Reply:
I wasn't able to find a way to only add a comment to `ignoreStatus` that is valid JSDOC. Can you point me to documentation that explains the proper syntax?

Reply:
It's just the same `/** */` comment style as this, but you add it to just a single field on the object. We've mostly done that for component props, but there's some places that we use it in this package like [this one](https://github.com/mattermost/mattermost/blob/master/webapp/platform/types/src/teams.ts#L57)

Reply:
Thanks for the hint. But I still can't get VSCode to show the comment.


![Screenshot from 2023-10-04 09-53-29](https://github.com/mattermost/mattermost/assets/16541325/85aad955-7693-4fa7-863b-c54c1ddd5cad)


![Screenshot from 2023-10-04 09-53-23](https://github.com/mattermost/mattermost/assets/16541325/fe37aafb-4f34-415b-ae80-456fa8669046)

It's surely unrelated to the PR, but do you know if any specific extension in needed?